AdyenComponents/PayTo/PayToComponent.swift

```diff
@@ -46,4 +58,75 @@ public final class PayToComponent: PaymentComponent,
         self.context = context
         self.configuration = configuration
     }
+
+    /// The payment flow selection title  label item.
+    internal lazy var flowSelectionTitleLabelItem: FormLabelItem = {
+        let item = FormLabelItem(
+            text: "How would you like to use Payto?",
```

Comment:
Let's add TODOs so we don't forget localization

Reply:
tbh not sure if we should merge this before we have the translation for it.
Otherwise the purpose of having a clean base branch is a bit defeated.

From my point of view the idea behind a base branch is that it's properly reviewed and ideally would not have to be re-reviewed before merging. By approving and merging TODOs and unfinished code we would still have to re-review the full base branch PR. 🤔

Reply:
Same counts for failing tests/workflows

Reply:
I have a suggestion:
- we do not block development as waiting for localization for a project when we build the train and the tracks at the same time can take time
- we add TODO
- we fail every PR to develop that has TODOs
- eventually we do not have long living TODOs in the code, they go to YouTrack instead

Reply:
Added TODO for string translation.